src/supermarket/app/controllers/slack_logs_controller.rb

```diff
@@ -0,0 +1,10 @@
+class SlackLogsController < ApplicationController
```

Comment:
We could call this "LogsController" to keep it around and only change its internal logic when the chat venue changes.

Reply:
"LogsController" makes me think this is going to have something to do with writing to logs, not reading things that are off-site.  "ChatController" might be a better option?

Reply:
Great point. 🤔

Reply:
Especially when the route is to `chat/`.